src/Analyzers/Core/Analyzers/RemoveRedundantEqualityWithTrue/RemoveRedundantEqualityWithTrueDiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.CodeAnalysis.RemoveRedundantEqualityWithTrue
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    internal sealed class RemoveRedundantEqualityWithTrueDiagnosticAnalyzer
```

Comment:
on the fence if i prefer `RemoveRedundantEqualityWithTrue` or just `RemoveRedundantEquality`.  on first pass, i prefer the latter.

Reply:
@CyrusNajmabadi If this is going to support both `== true` and `!= false`, will "RemoveRedundantEqualityOrInequality" make more sense?